docs/server/source/cloud-deployment-templates/ca-installation.rst

```diff
@@ -1,110 +1,108 @@
-city, org, email) are references and can be changed as per your
-deployment. Refer to the ``vars.example`` file for detailed variable descriptions.
+We customize the CA configuration by creating a ``vars`` file based
+on the existing ``vars.example`` file.
+The country, province, city, org and email values should be changed 
+to the values for your organization.
+Refer to the ``vars.example`` file for detailed variable descriptions.
 
-    .. code:: bash
+.. code:: bash
         
-        cd easy-rsa-3.0.1/easyrsa3
+   cd easy-rsa-3.0.1/easyrsa3
 
-        cp vars.example vars
+   cp vars.example vars
 
-        echo 'set_var EASYRSA_DN "org"' >> vars
-        echo 'set_var EASYRSA_REQ_OU "BigchainDB Deployment"' >> vars
-        echo 'set_var EASYRSA_KEY_SIZE 4096' >> vars
-        echo 'set_var EASYRSA_EXT_DIR "$EASYRSA/x509-types"' >> vars
+   echo 'set_var EASYRSA_DN "org"' >> vars
+   echo 'set_var EASYRSA_REQ_OU "BigchainDB Deployment"' >> vars
+   echo 'set_var EASYRSA_KEY_SIZE 4096' >> vars
+   echo 'set_var EASYRSA_EXT_DIR "$EASYRSA/x509-types"' >> vars
         
-        echo 'set_var EASYRSA_REQ_COUNTRY "DE"' >> vars
-        echo 'set_var EASYRSA_REQ_PROVINCE "Brandenburg"' >> vars
-        echo 'set_var EASYRSA_REQ_CITY "Berlin"' >> vars
-        echo 'set_var EASYRSA_REQ_ORG "BigchainDB GmbH"' >> vars
-        echo 'set_var EASYRSA_REQ_EMAIL "dev@bigchaindb.com" >> vars
+   echo 'set_var EASYRSA_REQ_COUNTRY "DE"' >> vars
+   echo 'set_var EASYRSA_REQ_PROVINCE "Berlin"' >> vars
+   echo 'set_var EASYRSA_REQ_CITY "Berlin"' >> vars
+   echo 'set_var EASYRSA_REQ_ORG "BigchainDB GmbH"' >> vars
+   echo 'set_var EASYRSA_REQ_EMAIL "dev@bigchaindb.com"' >> vars
 
-
-Modify the ``extendedKeyUsage = serverAuth`` to ``extendedKeyUsage =
-serverAuth,clientAuth`` in the file x509-types/server.
-Refer the MongoDB `documentation <https://docs.mongodb.com/manual/core/security-x.509/>`_ for more details on this.
+Edit the file ``x509-types/server`` and change
+``extendedKeyUsage = serverAuth`` to 
+``extendedKeyUsage = serverAuth,clientAuth``.
+See `the MongoDB documentation about x.509 authentication <https://docs.mongodb.com/manual/core/security-x.509/>`_ to understand why.
 
  
-Step 4: Create the Self-Signed CA
----------------------------------
+Step 4: Create a Self-Signed CA
+-------------------------------
 
-You can create the CA using:
+You can create a self-signed CA
+by going to the ``federation-ca/easy-rsa-3.0.1/easyrsa3`` directory and using:
 
-    .. code:: bash
+.. code:: bash
         
-        ./easyrsa init-pki
+   ./easyrsa init-pki
         
-        ./easyrsa build-ca
+   ./easyrsa build-ca
 
 
 You will be asked to enter a password for encrypting the ``ca.key`` file.
 Make sure to securely store this password. If you lose it, you risk not being
-able to add or remove entities from your PKI infrastructure in the future.
+able to add or remove entities in the future.
```

Comment:
Just `entities` is vague. What defines an entity?
And hence `entities from your PKI infrastructure` seems better, I think.